tests/test_wave_01.py

```diff
@@ -130,14 +131,15 @@ def test_update_task_not_found(client):
 
     # Assert
     assert response.status_code == 404
+    assert response_body == response.get_json()
```

Comment:
`response_body` references `response.get_json()` from what's already provided in the test on line 130. The assertion on line 134 is really just testing `response.get_json() == response.get_json()` where you're comparing something to itself. 

How could you refactor this assertion so that response_body is checked against an expected value for this scenario under test?

Reply:
✅ Did i add this correctly?

Reply:
I don't see a change here. Did you push something up?

I'd expect line 134 to be something like
```py
    assert response_body == {"message": "Task 1 is not found"}

Reply:
Thank you! Yes I pushed up I bunch of changes. Are they not going through?

Reply:
<img width="558" alt="Screenshot 2024-11-15 at 2 28 51 PM" src="https://github.com/user-attachments/assets/408a00b3-d8df-4522-b5c0-f0cf7ea629ad">

Reply:
![Uploading Screenshot 2024-11-15 at 7.15.23 PM.png…]()

Reply:
<img width="538" alt="Screenshot 2024-11-15 at 7 15 23 PM" src="https://github.com/user-attachments/assets/ec8fa430-3526-4361-8dbe-4ddf517d6ac4">